modules/client-configuration/pages/system-locking.adoc

```diff
@@ -14,43 +14,45 @@ System locks are implemented differently on traditional and Salt clients.
 
 When a traditional client is locked, no actions can be scheduled using the {webui}, and a padlock icon is displayed next to the name of the client in the menu:System[System List].
 
-To lock a traditional client using the {productname} {webui}, navigate to the [guimenu]``System Details`` page for the client you want to lock.
-Under [guimenu]``Lock Status``, click btn:[Lock this system].
 
+
+.Procedure: System Locking a Traditional Client
+. In the {productname} {webui}, navigate to the [guimenu]``System Details`` page for the client you want to lock.
+. Under [guimenu]``Lock Status``, click btn:[Lock this system].
 The client will remain locked until you click btn:[Unlock this system].
 
 //Something about requiring admin rights here maybe? --LKB 20200514
 
 Some actions can still be completed on locked traditional clients, including remote commands, and automated patch updates.
 To stop automated patch updates, navigate to the [guimenu]``System Details`` page for the client, and on the [guimenu]``Properties`` tab, uncheck [guimenu]``Auto Patch Update``.
 
-// All these webui things need to be checked, ref server is down right now. --LKB 20200514
-
 
 
 == System Locks on Salt Clients
 
 When a Salt client is locked, or put into blackout mode, no actions can be scheduled, Salt execution commands are disabled, and a yellow banner is displayed on the [guimenu]``System Details`` page.
 In this mode, actions can be scheduled for the locked client using the {webui} or the API, but the actions will fail.
 
+
 [NOTE]
 ====
 The locking mechanism is not available for Salt SSH clients.
 ====
 
-To lock a Salt client using the {productname} {webui}, navigate to the [guimenu]``System Details`` page for the client you want to lock.
-Navigate to the [guimenu]``Formulas`` tab, check the system lock formula, and click btn:[Save].
-Navigate to the menu:Formulas[System Lock] tab, check [guimenu]``Lock system``, and click btn:[Save].
-On this page, you can also specify specific Salt modules to enable them while the Salt client is locked.
-You do not need to apply the highstate.
 
+
+.Procedure: System Locking a Salt Client
+. In the {productname} {webui}, navigate to the [guimenu]``System Details`` page for the client you want to lock.
+. Navigate to the [guimenu]``Formulas`` tab, check the system lock formula, and click btn:[Save].
+. Navigate to the menu:Formulas[System Lock] tab, check [guimenu]``Lock system``, and click btn:[Save].
+On this page, you can also specify specific Salt modules to enable them while the Salt client is locked.
+. If instructed to by the {webui}, apply the highstate.
```

Comment:
```suggestion
. If instructed by the {webui}, apply the highstate.
```

Reply:
OK, so this is interesting. In my original form, I've split the infinitive verb "to apply". Removing the "to" would make the sentence grammatically incorrect, but what we can do is remove the split, which would make it something like "If instructed _to apply_ the highstate by the {webui}, do it". Which, quite frankly, sounds awful. I'm reminded of Orwell's advice to "break any of these rules before doing anything truly barbarous" :joy: I think the only way I can make this sound any good without the split infinitive is to make it more repetitive, like: "If the {webui} instructs you to apply the highstate, apply the highstate." :woman_shrugging: